packages/gatsby-theme-notes/src/components/directory-list.js

```diff
@@ -19,8 +19,8 @@ export default ({ directories }) =>
                 alignItems: `center`,
               }}
             >
-              <Folder style={{ marginRight: `10px` }} />
-              <span>{key}</span>
+              <Folder style={{ marginRight: `5px` }} />
+              <span style={{ marginRight: `15px` }}>{key}</span>
```

Comment:
should we move this to the link itself?

Reply:
Done.

Looks fine:

![Screenshot 2019-11-25 at 11 28 00 AM](https://user-images.githubusercontent.com/44570539/69532789-de65ea00-0f76-11ea-876f-92e3de006921.jpg)

Reply:
I meant moving the marginRight to the Link instead of adding it the span?

Reply:
Oh sure, gimme a second.

Reply:
Done as you suggested